CHANGES.md

```diff
@@ -9,6 +9,8 @@ Change Log
   * Removed `Camera.transform`, which was deprecated in Cesium 1.6. Use `Camera.lookAtTransform`.
   * Removed the `direction` and `up` options to `Camera.flyTo`, which were deprecated in Cesium 1.6. Use the `orientation` option.
   * Removed `Camera.flyToRectangle`, which was deprecated in Cesium 1.6. Use `Camera.flyTo`.
+* Deprecated
+  * The `smallterrain` tileset has been deprecated and will be removed in 1.11. Use the [STK World Terrain](http://cesiumjs.org/data-and-assets/terrain/stk-world-terrain.html) tileset instead.
```

Comment:
This is a new link to `cesiumjs.org` when all of the other links in this PR have been changed to `assets.agi.com`.  Is this correct?